examples/webgl_geometry_displacement.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - geometry - displacement</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				color: #808080;
+				font-family: Monospace;
+				font-size: 13px;
+				text-align: center;
+
+				background-color: #fff;
+				margin: 0px;
+				overflow: hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 100%;
+				padding: 5px;
+			}
+
+			a {
+
+				color: #0080ff;
+			}
+
+		</style>
+	</head>
+	<body>
+
+		<div id="container"></div>
+		<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - cpu displacement mapping</div>
+
+		<script src="../build/three.js"></script>
+		<script src='js/libs/dat.gui.min.js'></script>
+		<script src="js/controls/OrbitControls.js"></script>
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var gui = new dat.GUI();
+			var guiParams = { displacement: true };
+
+			var container, stats;
+
+			var loader = new THREE.TextureLoader();
+
+			var camera, scene, light, screenMesh, screenNonDisplaced, material, renderer;
+
+			var time = 0;
+			var emissiveMapped = false;
+
+			var controls;
+
+			var windowHalfX = window.innerWidth / 2;
+			var windowHalfY = window.innerHeight / 2;
+
+			var diffusePath = "textures/screen/diffuse.png";
+			var emissivePath = "textures/screen/emissive.png";
+			var heightPath = "textures/screen/height.png";
+			var normalPath = "textures/screen/normal.png";
+			var specularPath = "textures/screen/specular.png";
+
+			var reusableVector = new THREE.Vector3();
+
+			init();
+			animate();
+
+			function init() {
+
+				container = document.getElementById( 'container' );
+
+				camera = new THREE.PerspectiveCamera( 20, window.innerWidth / window.innerHeight, 1, 10000 );
```

Comment:
I would place the camera in the allowed range. So maybe:
```js
camera.position.set( 0, 0, 300 );
```
This will also avoid that the camera looks down to your object which is currently the case. The current position is `(0, 250, 0)` which is computed by `OrbitControls` in order to ensure the camera is between `minDistance` and `maxDistance`.